### PR TITLE
Keithley6221 - phase marker phase and line

### PR DESCRIPTION
**Pull request by AsafYagoda**
waveform_phasemarker_phase - A numerical property that controls the phase of the phase marker.
waveform_phasemarker_line - A numerical property that controls the line of the phase marker.

### CODE REVIEW

File: pymeasure/instruments/keithley/keithley6221.py [hunk 1]
```diff
@@ -224,6 +224,18 @@ def waveform_duration_set_infinity(self):
         values={True: 1, False: 0},
         map_values=True,
     )
+    waveform_phasemarker_phase = Instrument.control(
+        ":SOUR:WAVE:PMAR?", ":SOUR:WAVE:PMAR %s",
```
Comment (msmttchr) [inline pymeasure/instruments/keithley/keithley6221.py:228]:
```suggestion
        ":SOUR:WAVE:PMAR?", ":SOUR:WAVE:PMAR %g",
```
I guess this is a floating point number, so I propose `%g`, make sense ?

File: pymeasure/instruments/keithley/keithley6221.py [hunk 2]
```diff
@@ -224,6 +224,18 @@ def waveform_duration_set_infinity(self):
...
+        """ A numerical property that controls the phase of the phase marker.""",
+        validator=truncated_range,
+        values=[-180, 180],
+    )
+    waveform_phasemarker_line = Instrument.control(
+        ":SOUR:WAVE:PMAR:OLIN?", ":SOUR:WAVE:PMAR:OLIN %s",
```
Comment (msmttchr) [inline pymeasure/instruments/keithley/keithley6221.py:234]:
```suggestion
        ":SOUR:WAVE:PMAR:OLIN?", ":SOUR:WAVE:PMAR:OLIN %d",
```
Integer value ?

**Review by msmttchr — CHANGES_REQUESTED** [review]
@AsafYagoda, thank you very much for your contribution.
I submitted few comments, could You please have a look, if they make sense ?
Please add yourself also to AUTHORS.txt and authors.rst files.

**Review by msmttchr — APPROVED** [review]
Thanks for your contribution.

**Review by bilderbuchi — CHANGES_REQUESTED** [review]
LGTM after adding the author name as requested.

**Comment by AsafYagoda** [comment]
Added myself as a co-author as requested

**Comment by msmttchr** [comment]
> Added myself as a co-author as requested

@AsafYagoda, I don't see any change in the authors files ?

**Pull request merged**